test/unit/recipe/session/utils.unit.test.ts

```diff
@@ -0,0 +1,66 @@
+import { getFailureRedirectionInfo } from "../../../../lib/ts/recipe/session/utils";
+import * as utils from "supertokens-web-js/utils";
+
+beforeEach(() => {
+    jest.spyOn(utils, "getGlobalClaimValidators").mockReturnValue([
+        {
+            id: "id-3",
```

Comment:
These tests would be easier to read if the ids were a bit more descriptive - e.g.: `no-onFailureRedirection`, `onFailureRedirection-returns-undefined`, etc.